pkgs/games/xivlauncher/default.nix

```diff
@@ -3,7 +3,7 @@
 , useSteamRun ? true }:
 
 let
-  rev = "1.0.4";
+  rev = "1.0.5";
```

Comment:
can change to 1.0.6

Reply:
I'm not at my desktop right now but does deps.nix also need an update to build or is it using the same versions?

Reply:
Just checked locally since I was updating my overlay to manually build the new version until the PR is merged - fetch-deps produces an identical deps.nix for 1.0.5 and 1.0.6.